hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestBucketManagerImpl.java

```diff
@@ -67,51 +85,51 @@ private OzoneConfiguration createNewTestPath() throws IOException {
     return conf;
   }
 
-  private OmMetadataManagerImpl createSampleVol() throws IOException {
+  private void createSampleVol() throws IOException, AuthenticationException {
```

Comment:
Does this really need the AuthenticationException?

Reply:
We need that due to initialization of OzoneConfiguration. If I remove that there is an error from the IDE.